packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -3887,6 +3888,11 @@ export class WalletService {
         output.message = txpOut ? txpOut.message : null;
       });
       tx.customData = proposal.customData;
+
+      tx.createdOn = proposal.createdOn;
+      if (opts.includeExtendedInfo) {
```

Comment:
Maybe it's necessary to add `opts = opts || {};` at the beginning of the function?

Reply:
thanks. updated.